inflect/__init__.py

```diff
@@ -3649,13 +3649,12 @@ def ordinal(self, num: Union[int, Word]) -> str:
         else:
```

Comment:
d'oh: commit description should reference `ordinal` instead of `number_to_words`